### PR TITLE
[Readme Docs] Added Ruby installation note to Readme

### PR DESCRIPTION
**Pull request by Zain-Muiz**
Signed-off-by: Zain-Muiz <zainu10863@gmail.com>

**Description**

Added Installation note while installing ruby as beginners often tend to install the latest version of Ruby. ( which is different from that mentioned in Gemfile )

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ Before contributing, please review the [Contribution Flow](https://github.com/la
 
 - _The Meshery site is built using Jekyll - a simple static site generator! You can learn more about Jekyll and setting up your development environment in the [Jekyll Docs](https://jekyllrb.com/docs/)._
 
-- First [install Ruby](https://jekyllrb.com/docs/installation/), then install Jekyll and Bundler.
+- First [install Ruby](https://jekyllrb.com/docs/installation/), then install Jekyll and Bundler. ( **Note** : Install the same version of ruby as mentioned in the [Gemfile](https://github.com/meshery/meshery.io/blob/master/Gemfile) of this repository. )
```
Comment (warunicorn19) [inline README.md:57]:
```suggestion
- First [install Ruby](https://jekyllrb.com/docs/installation/), then install Jekyll and Bundler. ( **Note** : Install the same version of ruby as mentioned in the [Gemfile](https://github.com/meshery/meshery.io/blob/master/Gemfile) )
```
Consider this instead.

**Review by warunicorn19 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Zain-Muiz) [reply README.md:57]: Gotchya ! Alright !

**Comment by Zain-Muiz** [comment]
@warunicorn19 Done !

**Review by warunicorn19 — APPROVED** [review]
LGTM 🎉, Thanks @Zain-Muiz

**Review by leecalcote — APPROVED** [review]

**Pull request merged**